src/app/lib/ledger.ts

```diff
@@ -17,6 +18,20 @@ export async function canAccessNavigatorUsb(): Promise<boolean> {
   return await isSupported()
 }
 
+export async function canAccessBle(): Promise<boolean> {
+  let isSupportedOnMobile = false
+  try {
+    // Returns always true on web
+    isSupportedOnMobile = await BleTransport.isEnabled()
+  } catch (ex) {
+    /* ignore */
+  }
+  // Scan depends on requestLEScan method, which is not available on the web(feature flag)
+  const isSupportedOnDesktop = !!navigator?.bluetooth?.requestLEScan
```

Comment:
After I remove this so android app works, I can select a bluetooth ledger. But I still get WalletErrors.USBTransportNotSupported when I try to sign a transfer